src/mem.c

```diff
@@ -97,9 +97,10 @@ void tell_mem_status(char *nick)
 #ifdef DEBUG_MEM
   float per;
 
-  per = ((lastused * 1.0) / (memtbl_size * 1.0)) * 100.0;
-  dprintf(DP_HELP, "NOTICE %s :Memory table usage: %d/%d (%.1f%% full)\n",
-          nick, lastused, memtbl_size, per);
+  per = 100.0 * (lastused * 1.0) / (MEMTBLSIZE_MAX * 1.0);
+  dprintf(DP_HELP, "NOTICE %s :Memory table usage: %d/%d (%.1f%% of max "
```

Comment:
I don't see the benefit of STRINGIFY when we can just use %zu and print MEMTBLSIZE_MAX, then it doesn't need to be stringify'able

Reply:
it already is stringifyable. we dont have an extra variable here.